src/modules/private-share-folder/private-sharing.usecase.ts

```diff
@@ -70,4 +80,68 @@ export class PrivateSharingUseCase {
     );
     return folders;
   }
+
+  async stopSharing(folderUuid: Folder['uuid']): Promise<any> {
```

Comment:
Well, this is called from the controller so, as far as I have a JWT authorized and I know the folder `uuid` which I want to remove **I would be able to do so, even if it is not mine, even if I am not even related to this.** 

_Security is not an option, is a must._